sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/ExecutingSqlQueries_Net46.cs

```diff
@@ -190,8 +190,15 @@ public void Bar(SqlConnection connection, string param)
 
             command.CommandText = sensitiveQuery; // Noncompliant
 
+            string stillSensitive = sensitiveQuery;
```

Comment:
and what about 
```
string x = sensitiveQuery;
string y;
y = sensitiveQuery;
command.CommandText = stillSensitive;
```

Reply:
This is very similar with the test below: 
string x = null;
x = string.Format("INSERT INTO Users (name) VALUES (\"{0}\")", param);
command.CommandText = x; // FN

Reply:
it's not really the same...

my test is about propagation via simple assignment after a simple declaration

while your test is propagation via simple assignment after a declaration with inline initialization

it's 2 different test cases